cylc/flow/scripts/validate_install_play.py

```diff
@@ -55,10 +55,11 @@
 
 CYLC_ROSE_OPTIONS = COP.get_cylc_rose_options()
 VIP_OPTIONS = combine_options(
-    (['validate'], VALIDATE_OPTIONS),
-    (['install'], INSTALL_OPTIONS),
-    (['play'], PLAY_OPTIONS),
-    (['validate', 'install'], CYLC_ROSE_OPTIONS)
+    VALIDATE_OPTIONS,
+    INSTALL_OPTIONS,
+    PLAY_OPTIONS,
+    CYLC_ROSE_OPTIONS,
+    modify={'cylc-rose': 'validate, install'}
 )
```

Comment:
We're now hardcoding where options came from? Was the combination system unmanageable?

Reply:
It was giving me a massive headache - I think hardcoding it makes more sense, because it removes the need to store states outside the arguments/options objects.

Reply:
That's a shame, to make it easier you could `deepcopy(options_list)` which would allow you to modify these options in-place.

```python
if dest in option_list_a and dest in option_list_b
    this[SOURCES].add(option_b)
```